scripts/create-tables.js

```diff
@@ -0,0 +1,15 @@
+var spawn = require('cross-spawn');
+
+var create = spawn('psql', ['-d','webmaker_oauth_test', '-f', 'scripts/create-tables.sql']);
+
+create.stdout.on('data', function(data) {
```

Comment:
I think you could replace these pipes with the `stdio` option: https://nodejs.org/api/child_process.html#child_process_options_stdio (specifically the stdio: "inherit" option)